config/routes.rb

```diff
@@ -3,5 +3,9 @@
   devise_for :users
   root to: 'books#index'
   resources :books
-  resources :users, only: %i(index show)
+  resources :users do
+    resources :follows, only: %i(create destroy)
+    get 'followings' => 'follows#followings', as: 'followings'
+    get 'followers' => 'follows#followers', as: 'followers'
```

Comment:
```suggestion
    resources :follows, only: %i(create destroy) do
      get :followings
    end
```

みたいなルーティングを書く方がRailsっぽいかも。

https://railsguides.jp/routing.html#restful%E3%81%AA%E3%82%A2%E3%82%AF%E3%82%B7%E3%83%A7%E3%83%B3%E3%82%92%E3%81%95%E3%82%89%E3%81%AB%E8%BF%BD%E5%8A%A0%E3%81%99%E3%82%8B